humility-cli/src/lib.rs

```diff
@@ -158,7 +158,7 @@ impl ExecutionContext {
             if let Ok(e) = env::var("HUMILITY_PROBE") {
                 cli.probe = Some(e);
             } else if let Ok(e) = env::var("HUMILITY_IP") {
```

Comment:
Huh, this is news to me -- some commands use a _different_ env var for this, like `HUMILITY_RPC_IP`. Wonder how that happened.